src/Sql/dbo/Tables/OrganizationUser.sql

```diff
@@ -27,3 +27,8 @@ GO
 CREATE NONCLUSTERED INDEX [IX_OrganizationUser_OrganizationId]
     ON [dbo].[OrganizationUser]([OrganizationId] ASC);
 
+
+GO
+CREATE NONCLUSTERED INDEX [IX_OrganizationUser_StatusType]
+    ON [dbo].[OrganizationUser] ([Status], [Type])
+    INCLUDE ([OrganizationId], [UserId]);
```

Comment:
I am curious what sprocs would be using this index. Can you think of an area where we query org users by status and type? I can't think of any off hand.

Reply:
The following 2

```
OrganizationUser_ReadCountByOnlyOwner
OrganizationUser_ReadCountByFreeOrganizationAdminUser
```